core/src/main/scala/com/socrata/thirdparty/scalatra/RequestLogging.scala

```diff
@@ -0,0 +1,15 @@
+package com.socrata.thirdparty.scalatra
+
+import org.scalatra.ScalatraServlet
+import org.slf4j.LoggerFactory
+
+trait RequestLogging extends ScalatraServlet {
+  val logger = LoggerFactory.getLogger(getClass)
+  before() {
+    logger.info(request.getMethod + " - " + request.getRequestURI + " ? " + request.getQueryString)
```

Comment:
It's not. What convention do we want to use to name threads? I can do this in a separate PR.